README.md

```diff
@@ -29,6 +29,20 @@ npm install garden
 bower install garden
 ```
 
+### CDN
+
+CSS
+
+```
+<link href="https://unpkg.com/garden@0.34.0/dist/css/garden.min.css" rel="stylesheet">
+```
+
+JavaScript
+
+```
+<script scr="https://unpkg.com/garden@0.34.0/dist/js/garden.min.js"></script>
```

Comment:
I just knew about the @latest, that option without the version sounds much better